smart-contracts/hardhat.config.js

```diff
@@ -63,8 +63,10 @@ if (process.env.RUN_MAINNET_FORK) {
   }
   const alchemyURL = `https://eth-mainnet.alchemyapi.io/v2/${alchemyAPIKey}`
```

Comment:
Can we just replace this with the Unlock RPC?

Reply:
yes added in #10717